File: 프로그래머스/1/12901. 2016년/2016년.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(int a, int b) {
    string answer = "";
    string week[7] = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
    int month[12] = {0,31,29,31,30,31,30,31,31,30,31,30};
    int days = 0;
    
    for(int i = 0; i < a; i ++){
        days += month[i];
    }
    days += b - 1;
    answer = week[days % 7];
    return answer;
}

File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    answer = ''
    days = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    month = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    
    day_count = 0
    for i in range(0, a):
        day_count += month[i]
    answer = days[((day_count + b) % 7 ) - 1]
    
    return answer